src/components/ComponentDemos.js

```diff
@@ -1,51 +1,79 @@
-import { SearchAlt2 as Search } from '@styled-icons/boxicons-regular'
```

Comment:
Lets try to minimize the diff on this file -- refactor this page to show all the other demos and just scrollover for the data table 😀